Source/Atomic/UI/UIEditField.cpp

```diff
@@ -247,6 +260,22 @@ bool UIEditField::OnEvent(const tb::TBWidgetEvent &ev)
                 styleEdit->selection.SelectAll();
             }
         }
+
+#if defined(ANDROID) || defined( __ANDROID__) || defined(IOS)
+
+        // triple click to get the onscreen keyboard, in case it is auto-focused
+        else if ( ev.count == 3 ) 
+        {
```

Comment:
I am still not sure on the triple click, I will deploy to device and check this out :)